cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -7,11 +7,12 @@ severity "low"
 category "Cost"
 default_frequency "daily"
 info(
-  version: "5.6.1",
+  version: "5.6.2",
```

Comment:
@XOmniverse - does the meta parent policy template version also need to be updated (I know this policy and other policies listed here are deprecated)?

Reply:
@nia-vf1 It shouldn't in terms of the meta parents continuing to work, since the policy parameters and incident are unchanged.